modules/reindex/src/main/java/org/elasticsearch/index/reindex/BulkIndexByScrollResponse.java

```diff
@@ -152,7 +176,7 @@ public void readFrom(StreamInput in) throws IOException {
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         builder.field("took", took.millis());
         builder.field("timed_out", timedOut);
-        status.innerXContent(builder, params, false, false);
+        status.innerXContent(builder, new DelegatingMapParams(includeParams, params));
```

Comment:
Instead of using params maybe the caller can do this? Like, the REST endpoint knows about it. And it isn't a big deal to have the extra field in the task status.